server/app-engine/static/vehicle-assignments.js

```diff
@@ -340,22 +344,22 @@ function handleKey(id) {
         util.log('- fromDate: ' + fromDate);
         util.log('- toDate  : ' + toDate);
 
-        for (var item of items) {
+        for (item of items) {
             if (item.type === 'block' && item.status === 'assigned') {
-                var id = item.label;
-                util.log('-- id: ' + id);
+                var itemID = item.label;
```

Comment:
Please give a good look at this line - I'm renaming "id" to "itemID". Note that id is also a param for this function, and this change assumes we don't want that parameter value stored in the itemID variable. Is that correct?